### DISCUSSION TITLE
5 tests failed for the eclipse connector

### DISCUSSION THREAD

**Discussion by Metalesaek** [eclipse-edc/Connector#3367] [General]
I am building the eclipse connector  in   
windows 11.
Java version: 
openjdk 17.0.8 2023-07-18
OpenJDK Runtime Environment Temurin-17.0.8+7 (build 17.0.8+7)
OpenJDK 64-Bit Server VM Temurin-17.0.8+7 (build 17.0.8+7, mixed mode, sharing) 

Gradle version:
Gradle 8.2.1

However, the buid fail for the following  5 tests (copied from the index.html file):
•	[JsonLdExtensionTest](https://github.com/eclipse-edc/Connector/discussions/classes/org.eclipse.edc.jsonld.JsonLdExtensionTest.html). [verifyCachedDocsFromConfig_multipleEntries_oneIncomplete(ServiceExtensionContext, ObjectFactory)](https://github.com/eclipse-edc/Connector/discussions/classes/org.eclipse.edc.jsonld.JsonLdExtensionTest.html#verifyCachedDocsFromConfig_multipleEntries_oneIncomplete(ServiceExtensionContext,%20ObjectFactory))
•	[JsonLdExtensionTest](https://github.com/eclipse-edc/Connector/discussions/classes/org.eclipse.edc.jsonld.JsonLdExtensionTest.html). [verifyCachedDocsFromConfig_multipleEntries_oneInvalid(ServiceExtensionContext, ObjectFactory)](https://github.com/eclipse-edc/Connector/discussions/classes/org.eclipse.edc.jsonld.JsonLdExtensionTest.html#verifyCachedDocsFromConfig_multipleEntries_oneInvalid(ServiceExtensionContext,%20ObjectFactory))
•	[JsonLdExtensionTest](https://github.com/eclipse-edc/Connector/discussions/classes/org.eclipse.edc.jsonld.JsonLdExtensionTest.html). [verifyCachedDocsFromConfig_multipleValidEntries(ServiceExtensionContext, ObjectFactory)](https://github.com/eclipse-edc/Connector/discussions/classes/org.eclipse.edc.jsonld.JsonLdExtensionTest.html#verifyCachedDocsFromConfig_multipleValidEntries(ServiceExtensionContext,%20ObjectFactory))
•	[JsonLdExtensionTest](https://github.com/eclipse-edc/Connector/discussions/classes/org.eclipse.edc.jsonld.JsonLdExtensionTest.html). [verifyCachedDocsFromConfig_oneValidEntry(ServiceExtensionContext, ObjectFactory)](https://github.com/eclipse-edc/Connector/discussions/classes/org.eclipse.edc.jsonld.JsonLdExtensionTest.html#verifyCachedDocsFromConfig_oneValidEntry(ServiceExtensionContext,%20ObjectFactory))
•	[JsonLdExtensionTest](https://github.com/eclipse-edc/Connector/discussions/classes/org.eclipse.edc.jsonld.JsonLdExtensionTest.html). [verifyCachedDocsFromConfig_oneValidEntry_withSuperfluous(ServiceExtensionContext, ObjectFactory)](https://github.com/eclipse-edc/Connector/discussions/classes/org.eclipse.edc.jsonld.JsonLdExtensionTest.html#verifyCachedDocsFromConfig_oneValidEntry_withSuperfluous(ServiceExtensionContext,%20ObjectFactory))

I would appreciate any assistance to solve this issue.

**Comment by jimmarino** [comment]
We won't be able to diagnose the issue without proper stack traces or an indication of what went wrong.

**Comment by Metalesaek** [comment]
Thank you very much for your reply, I run it in Ubuntu 22.04 machine and it
worked fine. The problem now is that I am trying to build django project,
but I don't have any Clue how to set up The connector into it. Please if
you have some sources or use cases, I would appreciate it.
Thanks again.

**Comment by jimmarino** [comment]
Please review the documentation and samples.
